modules/govuk/manifests/node/s_backend_lb.pp

```diff
@@ -53,6 +53,7 @@
     'collections-publisher',
     'contacts-admin',
     'content-audit-tool',
+    'content-data-admin',
```

Comment:
I thought this was a frontend app (but I definitely could be wrong!). If so I think this should be in `s_frontend_lb.pp`.